katas/content/marking_oracles/palindrome/Placeholder.qs

```diff
@@ -0,0 +1,6 @@
+namespace Kata {
+    operation PalindromeOracle (input : Qubit[], target : Qubit) : Unit is Adj + Ctl {
```

Comment:
Let's aim for uniformity between the tasks from the MarkingOracles kata and the tasks migrated from other katas: use x and y as input and target names, and call oracles Oracle_Something (both here and in the rest of the tasks). Otherwise, switching from one naming scheme to another can be jarring, especially if someone copy-pastes some code!